common/models/load_balancer_summary.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+type LoadBalancerSummary struct {
```

Comment:
Main benefits in my eyes are:
* API contracts - keeping our current models for `GET /<entity>` and saying "oh, by the way, fields x, y, and z will always be blank for this call" is just plain lazy and confusing.
* Performance - this will vary by entity, but removing fields like `Dockerrun` for Deploys and `Deployments` for Services will speed things up significantly.